<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userNamangementWindow</class>
 <widget class="QWidget" name="userNamangementWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Management</string>
  </property>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameInput">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordInput">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="mainHeader">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Credentials</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="submitBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="status">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
